android/KMEA/app/src/main/assets/keyboard.html

```diff
@@ -56,6 +66,17 @@
       return oskWidth;
     }
 
+    function setIsChiral(m) {
+      var isChiral = false;
+      if (typeof m !== "undefined") {
```

Comment:
What is `m`? Can you use a clearer variable name?

Reply:
I'll rename to `keyboardID`